gitgud/test_user.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from gitgud.__main__ import GitGud
+
+def test_assert_initialized():
+    pass
+def test_handle_commit():
+    pass
+def test_handle_contrib():
+    pass
+def test_handle_debug():
+    pass
+def test_handle_explain():
+    pass
+def test_handle_goal():
+    pass
+def test_handle_help():
+    pass
+def test_handle_init():
+    pass
+def test_handle_issues():
+    pass
+def test_handle_level():
+    pass
+def test_handle_levels():
+    pass
+def test_handle_load():
```

Comment:
test load next/prev and test loading some generic level (or levels if you want).

Also test outside of an init-ed dir to make sure it doesn't throw an error